relay_messages_test.go

```diff
@@ -215,6 +222,38 @@ func withLazyCallResCombinations(f func(cr testCallRes)) {
 	}
 }
 
+func newCallResFrame(tb testing.TB, p testCallResParams) *Frame {
```

Comment:
similar to above, the call-req functions are defined a little differently,
```
func (cr testCallReq) frameWithParams(t testing.TB, p testCallReqParams) *Frame {
```

Should we do the same thing for consistency?

Reply:
i wanted to define this separately since the `.res()` calls are expected to return valid frames, while I'd also like to add tests for corrupted frames. I guess I could also add a `frameWithParams()` that wraps this, but it felt a bit redundant.